lib/sql_dust/utils/path_utils.ex

```diff
@@ -33,19 +34,19 @@ require IEx
   def prepend_path_aliases2(sql, options) when is_binary(sql) do
```

Comment:
Yes, sure, just wanted to keep it closer to your version to stay on the same page and discuss, before polishing it up.